src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/CreateNewListModal.tsx

```diff
@@ -76,6 +104,7 @@ export const CreateNewListModal: React.FC<CreateNewListModalProps> = ({
         }
       }
     `,
+    // updater: onListAdded,
```

Comment:
no need to check that for now, if we manage to get `@prependNode` directive to work we won't need this

Reply:
I think we should use the `updater` approach, since the user will have the opportunity to create a new list from the "[Select lists for this artwork](https://www.figma.com/file/wSV2KNz97Tyj8HkH2ZYAkB/Save-%2B-Lists?node-id=308%3A123937&t=3E13u44ClK91lojk-0)" modal (and for this reason it will be more difficult to get the `__id` from [connection](https://github.com/artsy/force/blob/f8e5c5fe4e5285cc7c08c96b4070c4d6cacf825d/src/Apps/CollectorProfile/Routes/Saves2/CollectorProfileSaves2Route.tsx#L90-L93) and further use of this id in `@prependNode`)

Reply:
amazing, then I am going full on the updater, thanks for looking into it

Reply:
Some investigations: https://github.com/artsy/force/commit/ef04456e28265e2775a88939292da72f4fdb9e36